### PR TITLE
Adding a general system overview dashboard

### PR DESCRIPTION
**Pull request by ben851**
# Summary | Résumé

Adding a dashboard that provides an overall system overview with some key information that I find myself looking for often (celery errors etc)

# Test instructions | Instructions pour tester la modification

Dashboard should deploy to staging and work correctly.

### CODE REVIEW

**Comment by sastels** [comment]
Do you have a link to an existing dashboard?

**Comment by ben851** [comment]
> Do you have a link to an existing dashboard you've been using?

Sure here it is in staging:
https://ca-central-1.console.aws.amazon.com/cloudwatch/home?region=ca-central-1#dashboards/dashboard/Ben-Notify-System

**Comment by sastels** [comment]
This is nice!
A couple thoughts
- can we make the send rates per minute rather than per 5 minutes (and add text (title maybe?) to reflect that)
- this seems to be a k8s dashboard essentially. Which would be good. We have [this one](https://ca-central-1.console.aws.amazon.com/cloudwatch/home?region=ca-central-1#dashboards/dashboard/Kubernetes) already which... isn't great. Maybe we could replace that one with this?

**Comment by sastels** [comment]
hrm, api lambda is on there to though, so maybe this really could be considered a system status dashboard.

**Comment by ben851** [comment]
> This is nice! A couple thoughts
> 
> * can we make the send rates per minute rather than per 5 minutes (and add text (title maybe?) to reflect that)
> * this seems to be a k8s dashboard essentially. Which would be good. We have [this one](https://ca-central-1.console.aws.amazon.com/cloudwatch/home?region=ca-central-1#dashboards/dashboard/Kubernetes) already which... isn't great. Maybe we could replace that one with this?

Made the metrics per minute as requested. The other idea, you answered your own question in your other comment :)

**Review by sastels — APPROVED** [review]
:shipit:

File: aws/common/dashboards.tf [hunk 1]
```diff
@@ -1,3 +1,338 @@
+resource "aws_cloudwatch_dashboard" "notify_system" {
+  count          = var.cloudwatch_enabled ? 1 : 0
+  dashboard_name = "Notify-System-Overview"
```
Comment (jimleroyer) [inline aws/common/dashboards.tf:3]: Is this dashboard specific to Kubernetes?

**Pull request merged**